app/src/main/res/values/strings.xml

```diff
@@ -365,12 +365,15 @@
     <string name="enable_smart_view">Enable smart screen</string>
     <string name="enable_smart_view_description">Smart screen shows relevant items instead of a grid</string>
     <string name="image_type_default">Default</string>
```

Comment:
Remove this string from the source file and keep the translations (will be auto-removed after merge)